src/util.js

```diff
@@ -8,3 +10,9 @@ export function extend(obj, props) {
 }
 
 
+/** Coerces and returns a lowercased representation of the argument.
+ *  @param {mixed} str
+ */
+export function lowercase (str) {
```

Comment:
hmm - this is how preact did lowercasing prior to 8.x, and it was removed for perf/size.  need to think on it a bit